modin/pandas/test/test_series.py

```diff
@@ -2602,7 +2551,7 @@ def h(x):
 
     def g(x, arg1=0):
         for _ in range(arg1):
-            x = x.append(x)
+            x = (pd if isinstance(x, pd.Series) else pandas).concat((x, x))
```

Comment:
same suggestion on concat-ing a tuple of `arg1 + 1` elements

Reply:
I'm not sure, your replacement is not equivalent to what it was before.